### ISSUE TITLE
Add option to exit out of "</>" mode in posting form

### ISSUE THREAD

**Issue by xieliansbignaturals** [BobaBoard/boba-components#152]
When a user clicks the "</>" button in the posting form, there's no way to click back out of it again to type normally. You have to completely discard the post and start again. I and a friend have tested this on Firefox mobile on Android, Firefox mobile on iOS, and Firefox desktop on iOS. On mobile (both versions), the bar from the screenshot below disappears, but on desktop, it stays and the "</>" icon is just unclickable.

We've agreed that it's possible there's a way to exit out that's obvious to people who would click this icon on purpose, but that it's still an issue that it's not clear to us two laypeople.

This is what we get stuck on:

<img width="561" alt="Grayed-out text box in posting form with popup bar showing various icons including </>" src="https://github.com/BobaBoard/boba-components/assets/157072966/04f81edf-23a9-4b84-890c-02bd0df813c1">

**Comment by essential-randomness** [comment]
Oh this is probably actually broken, but I would say it falls into the "the new editor fixes this" bucket. I don't have a fix for this off the top of my head, and I assume it doesn't happen often enough for it to be worth taking on urgently.

I'm assuming copying the text and pasting it again in the new message works? Or does the text need to be rewritten?

**Comment by xieliansbignaturals** [comment]
Yes, copying the text and pasting it again works just fine. I meant to include that, I just forgot! I definitely don't think this is urgent.

I'm sorry, I didn't know there was a new editor. Should I do something here? (There's a "close with comment" button?) I hope I didn't bother you too much! I wasn't sure whether to message you or put this here.

**Comment by essential-randomness** [comment]
It's absolutely fine to file something that ends up being marked as "not planned" and closed! It's still very valuable, even when that ends up being the case.

For example, people who follow this discussion now know we have a planned editor and that I'm hesitant to do work on it right now. Knowledge has been spread! It's also good that people see bug filing happening, because we need people to start doing it more and more (and soon with hopefully a team to help us sort and categorize these).

Also, you objective write pretty amazing and thorough bug reports. You show screenshots and reproduce issues. You're taking all the right steps. Please don't be discouraged from filing things! 

In short,
![giphy (1).gif](https://github.com/BobaBoard/boba-components/assets/47095486/eccdb66c-3633-4478-995f-bad1c734e6ba)

(Also, yes, closing the issue would be the right thing to do. IDK if you have the power to, but if you remind me tomorrow I'll give it to you :) )

**Comment by xieliansbignaturals** [comment]
Oh my gosh, thank you. 🥺❤️ This is such a reassuring response!! _How_ are you this nice and reassuring?? I was worrying so much - oh no! I did something wrong! - but I'm okay now.

I'll keep filing bug reports when I come across something, in that case (apart from stuff in the post editor). I really appreciate this.

I see a "close as not planned" option. I'll use that. <3
